testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/AbstractSinglenodeDeltaLakeDatabricks.java

```diff
@@ -46,18 +46,17 @@ public AbstractSinglenodeDeltaLakeDatabricks(Standard standard, DockerFiles dock
     @Override
     public void extendEnvironment(Environment.Builder builder)
     {
-        String databricksTestJdbcUrl = databricksTestJdbcUrl();
+        String databricksClusterId = databricksClusterId();
+        String databricksServerHostname = requireNonNull(System.getenv("DATABRICKS_SERVER_HOSTNAME"), "Environment DATABRICKS_LOGIN was not set");
+        String databricksOrganisationId = requireNonNull(System.getenv("DATABRICKS_ORGANISATION_ID"), "Environment DATABRICKS_LOGIN was not set");
```

Comment:
nit: Please fix error message.
```suggestion
        String databricksServerHostname = requireNonNull(System.getenv("DATABRICKS_SERVER_HOSTNAME"), "Environment DATABRICKS_SERVER_HOSTNAME was not set");
        String databricksOrganisationId = requireNonNull(System.getenv("DATABRICKS_ORGANISATION_ID"), "Environment DATABRICKS_ORGANISATION_ID was not set");
```